api/test/wfl/tools/endpoints.clj

```diff
@@ -7,7 +7,7 @@
   "The WFL server URL to test."
   (if (System/getenv "WFL_DEPLOY_ENVIRONMENT")
     "https://dev-wfl.gotc-dev.broadinstitute.org"
-    "http://localhost:8080"))
+    "http://localhost:3000"))
```

Comment:
do we still need to do this since #147 is merged in?

Reply:
I was thinking since these are API tests, it shouldn't require the UI server to be running so I changed it to use the API server's port directly. What do you think?

Reply:
I see, that totally makes sense!